docs/micropython.md

```diff
@@ -0,0 +1,77 @@
+# MicroPython on the BBC micro:bit
+
+Need to add some content here.
+- Thank all the MicroPython contributors
+
+Links:
+- https://microbit.org/get-started/user-guide/python/
+- https://github.com/microbit-foundation/micropython-microbit-v2
+- https://microbit-micropython.readthedocs.io/en/v2-docs/
+
+SoundEffect:
+- https://github.com/bbcmicrobit/micropython/pull/753
+- https://microbit-micropython--753.org.readthedocs.build/en/753/audio.html
+
+
+## Quick things to do with the micro:bit
```

Comment:
Yeah, that's a good point. When I was putting this list together it was because I thought it be useful to be a dense list of snippets for output stuff. The Reference and API sections are good, but I think random people at the table might quickly lose interest if they have to navigate through docs (including the reference section) to figure out how to quickly do something. Maybe the paragraph that links to do could just list the relevant sections in the Reference.